CHANGELOG/v1.7.0-rc.1.md

```diff
@@ -0,0 +1,457 @@
+- Control-plane: Allow mutation of all fields that should be mutable (#9871)
+- MachineDeployment: Add MachineSetReady condition to MachineDeployment (#9262)
+- MachineDeployment: Taint nodes with PreferNoSchedule during rollouts (#10223)
+- MachineHealthCheck: implement annotation to manually mark machines for remediation (#10202)
+- Release: Add dependencies section to release notes tool (#10043)
+- Release: Add release notes expander functionality (#10091)
+- Release: Add utility to create git issues on provider repo (#9110)
+- Testing: Resolve release markers (#9265)
+- Testing: Prepare quickstart, capd and tests for the new release including kind bump (#9872)
+
+## :bug: Bug Fixes
+- API: Use ptr instead of own implementation (#10276)
+- CAPD: Fix ignition to also set the kube-proxy configuration to skip setting sysctls (#9894)
+- CAPD: Remove --enable-hostpath-provisioner flag (#10271)
+- CAPD: Remove duplicate fix for btrfs/zfs support (#8376)
+- ClusterCacheTracker: Fix ClusterCacheTracker memory leak (#9543)
+- ClusterCacheTracker: Use RequeueAfter instead of immediate requeue on ErrClusterLocked to not have exponentially increasing requeue time (#9810)
+- ClusterClass: Fix for TestServerSideApplyWithDefaulting (#10307)
+- ClusterClass: Improve handling of topology orphaned objects (#10277)
+- clusterctl: Move handlePlugins function call out of init to allow debugging tests (#10200)
+- clusterctl: Validate no objects exist from CRDs before deleting them (#9808)
+- ClusterResourceSet: Requeue after 1 minute if ErrClusterLocked got hit (#9777)
+- Control-plane: KCP should defer remediation when a control plane machine is still provisioning (#9734)
+- Devtools: Show default cluster-template (#9820)
+- e2e: Add wait for MachineList to be available (#10301)
+- e2e: fix unset GINKGO_FOCUS variable (#10085)
+- e2e: Fix clusterctl upgrade e2e tests (enable CRS) (#10340)
+- e2e: Make MachinePools and MachineDeployments optional in ApplyClusterTemplateAndWait (#9960)
+- e2e: Re-introduce exclude capi-webhook-system to fix test flake (#10157)
+- e2e: Calculate correct worker count in clusterctl upgrade test (#9892)
+- e2e: Fix finalizers test to not only rely on namespaced name (#9891)
+- e2e: retry GetOwnerGraph in owner references test on certificate errors (#10201)
+- IPAM: Fix webhooks using mixed api versions (#9861)
+- KCP: Skip checking `clusterConfiguration.dns` fields when KCP checking MachineNeedRollout (#9857)
+- Logging: Improve log k/v pairs and a improve/drop a few log lines (#9813)
+- Machine: Bubble up machine drain condition in `MachinesReadyCondition` (#9355)
+- Machine: Watch external objects for machine before deleting (#10041)
+- MachinePool: Fix TestReconcileMachinePoolScaleToFromZero flakes (#9745)
+- API/e2e: Restore v1alpha3/v1alpha4 conversion to fix SSA issue & add e2e test coverage (#10147)
+- CABPK: Only refresh bootstrap token if needed, requeue in all cases where node hasn't joined yet (#9229)
```

Comment:
these should be relocated for alphabetical order

Reply:
Done, thanks! :)